java/src/jmri/util/TypeConversionUtil.java

```diff
@@ -0,0 +1,321 @@
+package jmri.util;
+
+import java.text.ParseException;
+import java.util.Collection;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import javax.annotation.CheckForNull;
+import jmri.Reportable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Converts between java types, for example String to Double and double to boolean.
+ * 
+ * @author Daniel Bergqvist Copyright 2019
+ */
+public final class TypeConversionUtil {
+    
+    /**
+     * Is this object a Boolean?
+     * @param object the object to check
+     * @return true if the object is a Boolean, false otherwise
+     */
+    public static boolean isBoolean(Object object) {
```

Comment:
Could you say a bit more about the motivation for this?  it's replacing a one-line common Java idiom with something JMRI specific.

Reply:
I don't remember but I think I added this method for completeness when I added the other similar methods. But I don't use it so I have removed it.